source/frontend/src/features/properties/map/compensation/detail/CompensationRequisitionDetailContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import { Api_AcquisitionFile } from 'models/api/AcquisitionFile';
+import { Api_Compensation } from 'models/api/Compensation';
+
+import { CompensationRequisitionDetailViewProps } from './CompensationRequisitionDetailView';
+
+export interface CompensationRequisitionDetailContainerProps {
+  compensation: Api_Compensation;
+  acquisitionFile: Api_AcquisitionFile;
+  clientConstant: string;
+  gstConstant: number;
+  loading: boolean;
+  setEditMode: (editMode: boolean) => void;
+  View: React.FunctionComponent<React.PropsWithChildren<CompensationRequisitionDetailViewProps>>;
+}
+
+export const CompensationRequisitionDetailContainer: React.FunctionComponent<
```

Comment:
Containers should have state, api interactions, or business logic. This component doesn't seem to do anything other then pass props and limit display. I think this can probably be removed. (it is fine if a view directly uses another view)